packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -69,9 +69,17 @@ export interface ClusterProps {
    * If true CloudWatch Container Insights will be enabled for the cluster
    *
    * @default - Container Insights will be disabled for this cluster.
+   * @deprecated See `containerInsightsLevel`
```

Comment:
```suggestion
   * @deprecated See {@link containerInsightsV2}
```

There are a few more references to `containerInsightsLevel`, I'm guessing that was refactored at some point?